ci/Jenkinsfile

```
@@ -70,16 +79,51 @@ pipeline {
   }
 
   stages {
-    stage('Download') {
+     stage('Download') {
       steps {
         script {
-          sh "mkdir '${env.WORKSPACE}/tmp/'"
-        }
-        fileOperations([
-          fileDownloadOperation(url:"${params.BUILD}", userName:"${CI_CREDS_USR}", password:"${CI_CREDS_PSW}", targetLocation:"./tmp", targetFileName:"StatusIm-Desktop.tar.gz", proxyHost:"", proxyPort:"")
-        ])
-        script {
-          sh "tar -zxvf '${env.WORKSPACE}/tmp/StatusIm-Desktop.tar.gz' -C '${env.WORKSPACE}/tmp'"
+          /* Creating directory for application build */
+          sh "mkdir -p '${env.WORKSPACE}/tmp/pkg/'"
+          def job_name = "${JOB_NAME}"
+          if(job_name.endsWith("nightly")){
+
+             /* Nightly regression */
+
+            if(params.BUILD_URL != ""){
```

You keep eating whitespace man.
```suggestion
            if (params.BUILD_URL != "") {
```

---

fixed